OpenSim/Actuators/MuscleFirstOrderActivationDynamicModel.h

```diff
@@ -85,8 +79,8 @@ namespace OpenSim {
 
     @author Matt Millard
 */
-class OSIMACTUATORS_API MuscleFirstOrderActivationDynamicModel : public Object{
-OpenSim_DECLARE_CONCRETE_OBJECT(MuscleFirstOrderActivationDynamicModel, Object);
+class OSIMACTUATORS_API MuscleFirstOrderActivationDynamicModel : public ModelComponent{
+OpenSim_DECLARE_CONCRETE_OBJECT(MuscleFirstOrderActivationDynamicModel, ModelComponent);
```

Comment:
I hadn't thought of that delineation, but rather separating Components that are used to assemble Models vs. providing purely computational services (like Operators, Solvers). I think it is OK to leave it as a ModelComponent. It is definitely a gray area.